pipinstaller/install_Functions.py

```diff
@@ -71,7 +70,7 @@ def installPip(printOut = False):
 #Function to install Package
 #######################################
 def installPackage(pkg, printOut = False):
-    cmd = f"pip install {pkg[1]} --user"
+    cmd = "py -3 -m pip install {} --user".format(pkg[1]) if sys.platform == 'win32' else "python3 -m pip install {} --user".format(pkg[1])
```

Comment:
same as above, feel f-string might be cleaner.
```python
cmd = f"{pycmd} -m pip install {pkg[1]} --user"
```
I agree though that it should change

Reply:
F strings are not supported in python 3.5 and below
[https://www.python.org/dev/peps/pep-0498/](https://www.python.org/dev/peps/pep-0498/)
See effadfc